tests/e2e/set-ffi-env-e2e

```diff
@@ -273,6 +273,29 @@ then
     exit
 fi
 
+echo
+info_message "Checking if QM already installed"
+info_message "=============================="
+QM_STATUS="$(systemctl is-enabled qm 2>&1)"
```

Comment:
Can we use if_error_exit instead ? Seems better to get error code than output text to my eyes. WDYT?

Reply:
Ack